### DISCUSSION TITLE
Are notebooks and cli still supposed to be maintained?

### DISCUSSION THREAD

**Discussion by mspronesti** [sinaptik-ai/pandas-ai#523] [General]
[Dev discussion] Looks like that both the cli and the notebooks haven't been maintained for a while.
Are they supposed to be removed in a future release ?

**Comment by gventuri** [comment]
@mspronesti very good points.

CLI: I think it might still make sense, even tho I'm wondering if someone is even using it, as I haven't seen anyone complaining about the CLI not working.

Notebooks: here the real question is more about "do we want to provide examples as .py files or use the notebooks instead"? If we opt in for the notebooks, maybe we could rewrite all the examples as notebooks, while if we decide to use .py files for the examples, we can remove the notebooks. What do you think?
